datasets/wikitext_tl_39/wikitext_tl_39.py

```diff
@@ -0,0 +1,112 @@
+"""TODO(wikitext): Add a description here."""
+
+from __future__ import absolute_import, division, print_function
+
+import os
+
+import datasets
+
+
+# TODO(wikitext): BibTeX citation
+_CITATION = """\
+@article{cruz2019evaluating,
+  title={Evaluating Language Model Finetuning Techniques for Low-resource Languages},
+  author={Cruz, Jan Christian Blaise and Cheng, Charibeth},
+  journal={arXiv preprint arXiv:1907.00409},
+  year={2019}
+}
+"""
+
+# TODO(wikitext):
+_DESCRIPTION = """\
+Large scale, unlabeled text dataset with 39 Million tokens in the training set. Inspired by the original WikiText Long Term Dependency dataset (Merity et al., 2016). TL means "Tagalog." Originally published in Cruz & Cheng (2019).
+"""
+_URL = "https://github.com/jcblaisecruz02/Filipino-Text-Benchmarks"
+_LICENSE = "GPL-3.0"
+_DATA_URL = "https://s3.us-east-2.amazonaws.com/blaisecruz.com/datasets/wikitext-tl-39"
+
+
+class WikitextTL39Config(datasets.BuilderConfig):
+    """BuilderConfig for WikiText."""
+
+    def __init__(self, data_url, **kwargs):
+        """BuilderConfig for Wikitext
+
+        Args:
+          data_url: `string`, url to the dataset (word or raw level)
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(WikitextTL39Config, self).__init__(
+            version=datasets.Version(
+                "1.0.0",
+            ),
+            **kwargs,
+        )
+        self.data_url = data_url
+
+
+class WikitextTL39(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class WikitextTl39(datasets.GeneratorBasedBuilder):
```
This must be the camel case version of the dataset script name.

Can you also move the dataset directory from `./datasets/wikitext_tl_39` to `./datasets/wikitext_tl39` instead, and rename the script accordingly ? This way the camel case and snake case names match.

Reply:
Gotcha! I'll work on the changes then start a new branch. Thanks!